packages/vaadin-login/src/vaadin-login-mixin.d.ts

```diff
@@ -31,6 +31,11 @@ interface LoginMixin {
    */
   noForgotPassword: boolean;
 
+  /**
+   * If set, the user name field does not automatically receive focus when the component is attached to the document.
```

Comment:
Please add the attribute annotation to make `lit-analyzer` happy:

```
@attr {boolean} enter-next-row
```

Reply:
Done